src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -37,7 +38,7 @@ class Post911GIBStatusApp extends React.Component {
           verifyUrl={this.props.verifyUrl}>
         <AppContent>
           <div className="row">
-            <div className="usa-width-two-thirds medium-8 columns">
+            <div className="small-12 columns">
```

Comment:
I had to swap where some of the classes are defined so that the system-down and no-records errors span the full width.